packages/sdk-react/src/components/products/ProductDeleteModal/ProductDeleteModal.tsx

```diff
@@ -65,7 +65,7 @@ export const ProductDeleteModal = ({
         <Button
           variant="outlined"
           color="error"
-          disabled={deleteProductMutation.isLoading || isInitialLoading}
+          disabled={deleteProductMutation.isPending || isInitialLoading}
```

Comment:
`isInitialLoading` is [deprecated](https://tanstack.com/query/latest/docs/framework/react/guides/migrating-to-v5#:~:text=is%20deprecated%20now%20and%20will%20be%20removed%20in%20the%20next%20major%20version). Would be cool to remove its usage globally.

Reply:
fixed 

I saved the logic according to tanstack query v5 guide
```
isLoading -> isPending
isInitialLoading -> isLoading
```
![image](https://github.com/team-monite/monite-sdk/assets/6762217/4dd8d3e4-1058-4035-a98a-270d8e28dfda)